test/org/sagebionetworks/bridge/validators/SurveyAnswerValidatorTest.java

```diff
@@ -125,10 +125,27 @@ public void validateDecimalMinValue() {
     @Test(expected = InvalidEntityException.class)
     public void validateDecimalMaxValue() {
         DecimalConstraints constraints = new DecimalConstraints();
+        constraints.setMaxValue(10d); 
+        validator = new SurveyAnswerValidator(createQuestion(constraints));
+        Validate.entityThrowingException(validator, createAnswer("15.0"));
+    }
+    @Test(expected = InvalidEntityException.class)
+    public void validateInvalidDecimalStep() {
```

Comment:
An interesting case for that would be also allowing multiple values, and multiple values are out of range... I'll add that.